### ISSUE TITLE
High resolution screenshot (low-priority)

### ISSUE THREAD

**Issue by oliviertassinari** [argos-ci/argos#1089]
Right now, Argos CI is configured to handle images that are 1 device px = 1 CSS pixel. I wonder if it would make sense in the future to support higher-resolution screenshots.

I can't find any screenshot tool supporting this, but I suspect that it's trending toward becoming the industry norm (meaning we will start to see budget Windows option support this in the next 5 years).

**Comment by gregberge** [comment]
Hey @oliviertassinari, I think you can already do it using Playwright (https://playwright.dev/docs/api/class-browser#browser-new-context-option-device-scale-factor).

**Comment by oliviertassinari** [comment]
Yes, but once the screenshot is displayed in the UI of Argos, it's displayed too big.

**Comment by gregberge** [comment]
@oliviertassinari still needed?
